gst-plugin/src/drpai-models/base_post_processor.cpp

```diff
@@ -0,0 +1,115 @@
+//
+// Created by matin on 03/11/24.
+//
+
+#include "base_post_processor.h"
+#include <fstream>
+
+/// Renders bounding boxes on the image using the latest detections.
+/// This function usually gets overridden by the child class. But it is not a must.
```

Comment:
I'm thinking if it should also mention where the bounding box coordinates originate from also since they're not part of the argument list, I want your comment about this, too though